app/src/marketplace/components/Modal/DeployingCommunityDialog/index.jsx

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+import React from 'react'
+import { I18n, Translate } from 'react-redux-i18n'
+import cx from 'classnames'
+
+import Modal from '$shared/components/Modal'
+import Dialog from '$shared/components/Dialog'
+import { type Product } from '$mp/flowtype/product-types'
+
+import styles from './deployingCommunityDialog.pcss'
+
+export type Props = {
+    product: Product,
+    onClose: () => void,
+    onContinue: () => void,
+}
+
+// $FlowFixMe
+const formatSeconds = (seconds) => new Date(seconds * 1000).toUTCString().match(/\d\d:(\d\d:\d\d)/)[1]
```

Comment:
```js
// 1 hour
formatSeconds(60 * 60) // 00:00
```

Reply:
I guess it's unlikely that the estimate is over an hour but I added the hours in as well.